### DISCUSSION TITLE
Using 'n' ends up searching for the last searched term and thus moving the cursor

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [nvim-tree/nvim-tree.lua#2153] [Q&A] [answered]
Whenever I use <kbd>n</kbd> as part of the default keybinds, my cursor usually ends up jumping around as it triggers searching for the next instance of the last searched term. I'm sure I'm missing some fairly obvious workaround here but could someone please be patient enough to point out what it might be?

**Comment by gegoune** [comment]
This is default vim/nvim's mapping. What do you expect to happen instead?

**Reply by lokesh-krishna** [reply]
A way to disable it in the nvim-tree buffer? How does one handle the cursor jumping around. I hit <kbd>n</kbd><kbd>a</kbd> to create a new file at one place but the <kbd>n</kbd> moves the cursor elsewhere and I get the prompt to create the file at a different location than the one I intended.

**Reply by gegoune** [reply]
Why do you press `n` for exactly? It does what is suppose to do, go to next match of last searched pattern.

**Reply by lokesh-krishna** [reply]
Um, the keybind to create a new file is <kbd>n</kbd><kbd>a</kbd>, right? So when I'm pressing <kbd>n</kbd> for that is when I don't want it to move to the next match.

**Reply by gegoune** [reply, marked as answer]
No, it's not, it's just `a` by default. Are you mistaking `n` from `vim.keymap.set`'s first argument (which is mode, see `:h vim.keymap.set`) with actual mapping?

In short, just use `a`. `n` is for `normal` mode.

**Reply by lokesh-krishna** [reply]
Wow, I feel stupid. Thanks for being so patient!
